CADD.pm

```diff
@@ -93,7 +93,6 @@ sub run {
   
   # get allele, reverse comp if needed
   my $allele = $tva->variation_feature_seq;
```

Comment:
@ima23 - this works fine, but could you update the comment.

Reply:
Thanks @seh, I've updated it.